mmv1/third_party/terraform/services/cloudfunctions2/data_source_google_cloudfunctions2_function_test.go

```diff
@@ -24,9 +24,11 @@ func TestAccDataSourceGoogleCloudFunctions2Function_basic(t *testing.T) {
 			{
 				Config: testAccDataSourceGoogleCloudFunctions2FunctionConfig(functionName,
 					bucketName, zipFilePath),
+				// As the value of "labels" and "terraform_lables" in the state is dpendent on the configuration,
+				// and these fields are not set in the configuration of the data source, so these fields are empty in the state of the data source.
```

Comment:
Oh, that's a good point that I didn't consider. Time-permitting we should try to correct that- we could copy the value of `effective_labels` on top of `labels` in the datasource's Read function post-resource Read. We could always fix that in a minor or patch release, though.

Reply:
Thanks for the idea. I will test and handle all of the relevant data sources later.